cves/kernel/CVE-2021-34556.yml

```diff
@@ -317,8 +322,9 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: "net"
+  note: |
+    Looked at path for the changed files
 interesting_commits:
```

Comment:
If there is no interesting commits, then there should be an explanation of why. Rudimentary fixing? Single commit fix?